### PR TITLE
created PopUp common-component and stayled it relate #25

### PR DESCRIPTION
**Pull request by someyaaltous**
Create the PopUP component on Common Components and give it a style using the styled-component

### CODE REVIEW

**Review by sara219 — APPROVED** [review]

**Review by SajaLahaleeh — APPROVED** [review]

File: src/CommonComponents/Popup/index.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+import {
+  PopUPCard, PopUpPox, Logo, LogoImage, BodyPage,
+} from './index.style';
+
+const PopUp = () => (
+  <BodyPage>
+    <PopUPCard>
+      <PopUpPox>
+        <Logo>
+          <LogoImage src={popuplogo} alt="mainlogo" />
```
Comment (95Rawan) [inline src/CommonComponents/Popup/index.js]: alt should be "a kid holding a kite logo"

File: src/CommonComponents/Popup/index.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+import { PopUPCard, PopUpPox, Logo, LogoImage, BodyPage ,PopUpDiv,LeftButtonPopUp,RightButtonPopUp,TitleP } from './index.style';
+import popUpdata from '../../Data/popupdata'
+class PopUp extends React.Component {
+  state = {
+    title: '',
+    descraption: '',
+    leftButton: '',
+    rightButton: ''
+  };
+  render() {
```
Comment (hshahwan) [inline src/CommonComponents/Popup/index.js]: I think this component will not be able to used in different places, so please change it to stateless component and pass props to help you to use in different places

File: src/Data/popupdata.js [hunk 3]
```diff
@@ -0,0 +1,10 @@
+const popUpdata = [{
```
Comment (hshahwan) [inline src/Data/popupdata.js]: no need for this file

File: src/CommonComponents/Popup/index.style.js [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+  background: #cfcfcf;
+  margin: -1.09rem;
+`;
+export const PopUPCard = styled.div`
+  display: flex;
+  justify-content: center;
+  margin-top: -21rem;
+`;
+export const PopUpPox = styled.div`
+  place-content: end center;
```
Comment (hshahwan) [inline src/CommonComponents/Popup/index.style.js]: you can use flexbox properties 

```  justify-content: center
align-items: flex-end
```

Comment (hshahwan) [inline src/CommonComponents/Popup/index.style.js]: use flexbox properties 
```
justify-content: center;
align-items: flex-end;
```

**Review by sara219 — APPROVED** [review]

**Pull request closed without merging**